pkg/kubelet/kubelet_pods.go

```diff
@@ -146,6 +147,11 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 
 			hostPath = filepath.Join(hostPath, mount.SubPath)
 
+			err = volumevalidation.ValidatePathNoBacksteps(hostPath)
```

Comment:
this is still outstanding... this needs to be moved above the filepath.Join line above